pilot/cmd/pilot-agent/status/server.go

```diff
@@ -788,6 +788,7 @@ func (s *Server) handleAppProbeHTTPGet(w http.ResponseWriter, req *http.Request,
 	}
 	// We only write the status code to the response.
 	w.WriteHeader(response.StatusCode)
+	_, _ = io.Copy(w, response.Body)
```

Comment:
Do we need the body drain above if we copy it here?

Reply:
I meant that there is no need to drain the body after this copy and just close it.
But it might not be harmful.

Reply:
I've change to use utilio.ReadAtMost(response.Body, maxRespBodyLength) instead.